client/scss/components/_listing.scss

```diff
@@ -247,6 +241,11 @@ ul.listing {
         }
       }
     }
+
+    .icon-folder {
+      margin: 3px 0.3em 0 0;
+      vertical-align: top;
+    }
```

Comment:
This was originally `.icon.initial` in 6.0:
https://github.com/wagtail/wagtail/blob/b5e068e6a64bec98d7670c00f39d92fe3a3d22db/client/scss/components/_listing.scss#L247-L250

but was changed to `.icon-folder` in d04746bf9ed43928791a23485193e8d71508ad82. As a result, you'll see that the globe icon for site roots is no longer aligned:

## This PR

![image](https://github.com/wagtail/wagtail/assets/6379424/5d0f599a-e250-40f9-968e-c53a3d2077fb)

## 6.0
![image](https://github.com/wagtail/wagtail/assets/6379424/1406fdef-cb86-48c7-b8ff-01d754e6dcb3)